project/roadmap.md

```diff
@@ -4,26 +4,28 @@
 
 If you are interested in collaborating on any of the roadmap features or other features in the project, please mail [Mbed TLS](https://lists.trustedfirmware.org/mailman/listinfo/mbed-tls) or [PSA Crypto](https://lists.trustedfirmware.org/mailman/listinfo/psa-crypto) mailing lists.
 
-**2024 CQ2 (In Development)**
-
-* [Mbed TLS] Mbed TLS4.0 Preparations & Planning
+**2024 CQ3 (In Development)**
+* [Mbed TLS] [Mbed TL3.6.1](<https://github.com/orgs/Mbed-TLS/projects/1#column-19883226>)
+* [PSA Crypto] [PSA Crypto Repo split foundation1](<https://github.com/orgs/Mbed-TLS/projects/1#column-19914196>)
+* [PSA Crypto] [PSA Crypto Repo split foundation2]([<https://github.com/orgs/Mbed-TLS/projects/1#column-19914196>)
+* [PSA Crypto] [PSA Crypto Repo split foundation3](<https://github.com/orgs/Mbed-TLS/projects/1#column-19868513>)
+* [Mbed TLS] [Mbed TLS4.0 Preparations1](<https://github.com/orgs/Mbed-TLS/projects/1#column-19890139>)
+* [Mbed TLS] [Mbed TLS4.0 Preparations2](<https://github.com/orgs/Mbed-TLS/projects/1#column-19886039>) 
 * [Mbed TLS] [Bignum Improvements](<https://github.com/orgs/Mbed-TLS/projects/1#column-19853826>)
```

Comment:
All the issues are in -restricted, so non-team members just see a column with no meaningful information. We should either not list it or explain what it is.

Reply:
ok. Removed link. Kept Bignum Improvements as an entry still